activities/communication/communications-plan.md

```diff
@@ -0,0 +1,169 @@
+---
+type: resource
+---
+
+# Communications plan
+
+The Foundation for Public Code relies on communications to [achieve its mission](../../organization/mission.md).
+
+## Principles
+
+Our communications need to reflect our core values: open, quality, trust, community, helpful. They must also be non-partisan, in line with our [founding principles](../../organization/mission.md#founding-principles).
+
+Our messages range from neutral in tone to positive and inspirational. Our community won’t be sustainable if we attract people motivated by fear or anger.
+
+Our communications are in plain English, but we link to highly relevant pages in other languages, for example about our members’ work.
+
+Our communications should show to showcase the Foundation for Public Code as:
+
+* active, dynamic, getting things done
+* smart enough to be trusted with the importance and complexity of its work
+* plugged into and supportive of the network of visionaries and activists who also want this future
+* open and transparent, both about its intentions and operations as well as learnings, failures and successes
+
+We believe it’s more important to talk concretely about what we’ve already done than what we hope to do.
+
+## Objectives
+
+Our communications should help us achieve [our current priorities](../../organization/mission.md#current-priorities).
+
+## Audience
+
+Our audience is divided into:
+
+* current Foundation contributors
+* future Foundation contributors
+* influencers
+
+### Foundation contributors
+
+We believe [these are potential contributors to the Foundation for Public Code](https://about.publiccode.net/activities/value-and-impact/user-mapping/).
+
+### Influencers
+
+These are domain celebrities in open source, government digital transformation and civic tech.
+
+## Unique selling proposition
+
+We are the only organization focused on open source software for public organizations that:
```

Comment:
This list would make more sense if it was clear that not any of these but all of these together make us what we are, as all of them are done by others.